src/commands/piracy-acts.ts

```diff
@@ -0,0 +1,170 @@
+export const PiracyActs = [
```

Comment:
small note: this is also updated to match what's on the website https://spellboundgames.co.uk/pages/one-piece-bounty-board - see our Discord DMs for more details